Sources/MenuAnimator.swift

```diff
@@ -122,8 +122,7 @@ class MenuInteractiveTransition: NSObject, UIViewControllerInteractiveTransition
     private var panRecognizer: UIPanGestureRecognizer!
```

Comment:
Ok, thank you.
I'm going to remove all implicitly unwrapped optionals in all files, but in the develop branch, when this branch is merged. Please, not consider this right now. I'd like to do refactoring implicitly unwrapped optionals within a single commit.

Here I'm going to add the possibility to update user's animation settings after the device rotation.